lib/gr_blocks/nonstop_wavfile_sink_impl.cc

```diff
@@ -267,7 +267,7 @@ int nonstop_wavfile_sink_impl::dowork(int noutput_items,  gr_vector_const_void_s
   //long curr_src_id = 0;
 
   for (unsigned int i = 0; i < tags.size(); i++) {
-    if (pmt::eq(this_key, tags[i].key) && d_current_call->get_system_type() == "conventionalP25") {
+    if (pmt::eq(this_key, tags[i].key) ) {
```

Comment:
I think this could lead to trouble. Sometimes the Control Channel is slightly ahead of the decoding on the Voice Channel, so there maybe a lot of oscillation between source IDs. Is there a particular reason for making this change? Are source IDs different between the 2 channels?

Reply:
This addressed #200 for me, with IDs not showing on the control channel. If no one else is experiencing the problem, this can be skipped. As I noted, what I'd like to do is log the control channel and voice frame sources separately (I'm envisioning two separate srcLists in the JSON files).

Reply:
I want to dig into this more before adding in this change for everybody. Taking in Source IDs for a recording in 2 different spots seems like trouble. Could it be that some of the control channel messages are being processed incorrectly? Another alternative would be to have all of the Source IDs come from the Voice Channel. This would probably make the Source IDs line up better with the recordings.

Let's leave this change out of the PR and add to items to the list:
 - investigate why no Source IDs are in the Control Channel Messages
 - Look to switching all Source IDs to come from the Voice Channel.

Reply:
As I said in my original comment, "I'd like to have radio IDs from the control channel and voice frames logged to 2 separate lists". Given that this was really just a quick fix for me, I'm definitely willing to leave it out.

[This system](https://openmhz.com/system/osrppdxc) is also using this code, but so much of the calls are patched from elsewhere.